Bottom: a4d87fda3c02362c6f1680c2b3b4d9408a488caa
Top:    27db12b0845f1cb61171bd3247bf3a5217cf9a9d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-02 14:26:57 +0000

lib: Fix make distcheck

The generated newlib files need to be removed by make distclean


---

diff --git a/lib/Makefile.am b/lib/Makefile.am
index 95be997..d5a62f1 100644
--- a/lib/Makefile.am
+++ b/lib/Makefile.am
@@ -57,6 +57,8 @@ ${NEWLIB}.stamp : footprint.pcb ${NEWLIB_DEPS}
 		${PREVIEW} --pcb "${PCB}" --awk "${AWK}"
 	echo '${PREVIEW_MSG}' > $@
 
+distclean-local:
+	rm -fr ${NEWLIB}
 
 install-data-local: ${NEWLIB}.stamp
 	if test -d ${NEWLIB} ; then \
